ansible/roles/azure/templates/dashboardTemplate.json

```diff
@@ -0,0 +1,1312 @@
+{
```

Comment:
I'm curious how this file was generated.  Was a dashboard created in a UI and then exported?

Reply:
I'm pretty sure this was originally created from UI and then exported as a template. There is [public documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/template-workspace-configuration) on the ARM template schema for such templates but it is much easier to start with an exported template and then trim it down.

Reply:
Yes correct - the approach is documented [here](https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-dashboards-create-programmatically)

Reply:
Also to pre-empt a likely question, the template schema for the "workbooks" is described [here](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/workbooks-automate). Think of workbooks as a drilldown into each of the "tiles" that the dashboard provides.